app/controllers/users/notification_settings_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Users
+  class NotificationSettingsController < ApplicationController
```

Comment:
**blocking** There are a couple of actions missing that must be borrowed from `UsersController` (GitHub is having some issues so I can't link the permalink):

- `before_action :raise_suspended` this should be copied over, we don't want suspended users to be able to modify their data
- `after_action :verify_authorized` we need this and the corresponding call to authorize the current user, we don't want rhymes to be able to update Arit's settings

I believe you can recycle `UserPolicy` here, with something like: `authorize current_user, policy_class: UserPolicy` as the first statement in the `update` action